config/database.yml

```diff
@@ -1,16 +1,18 @@
-development:
+default: &default
   adapter: postgresql
-  database: growstuff_dev
-  host: localhost
+  encoding: unicode
+  host: db
```

Comment:
In my last job, we had *two* "setup environment" processes, and skew between the two was a constant nightmare - people using the old setup kept unwittingly breaking the new one, which meant that the new setup was constantly playing catch-up and could never *quite* take over. So I think we *should* aim to have a clean switchover if possible, rather than trying to maintain "automated environment setup" and "manual environment setup" as equal first-class citizens.